README.md

```diff
@@ -24,9 +24,9 @@
 ![OpenVINO XAI Concept](docs/source/_static/ovxai-concept.svg)
 
 **OpenVINO™ Explainable AI (XAI) Toolkit** provides a suite of XAI algorithms for visual explanation of
-[**OpenVINO™**](https://github.com/openvinotoolkit/openvino) Intermediate Representation (IR) models.
+[**OpenVINO™**](https://github.com/openvinotoolkit/openvino) as well as [**PyTorch**](https://pytorch.org) and [**ONNX**](https://onnx.ai/) models.
 
-Given **OpenVINO** models and input images, **OpenVINO XAI** generates **saliency maps**
+Given AI models and input images, **OpenVINO XAI** generates **saliency maps**
```

Comment:
@negvet We use the exact same sentence in all of the notebooks and all of their READMEs, so we shouldn't forget to update it as well